torch/_dynamo/symbolic_convert.py

```diff
@@ -2035,15 +2035,14 @@ def inline_call_(
         try:
             sub_locals, closure_cells = func.bind_args(parent, args, kwargs)
         except TypeError as e:
-            log.warning(
-                "%s %s %s %s %s",
-                func.get_filename(),
-                func.get_function(),
-                args,
-                kwargs,
-                e,
+            raise Unsupported(
+                "{reason}.\n  func = {func}, args = {args}, kwargs = {kwargs}".format(
```

Comment:
Mmm, I don't think returning `func` `args` `kwargs` is that helpful. Maybe look up frame summaries and current loc from the tracing context instead.

Reply:
The error here is because of signature mismatch. I'm not sure why it's not helpful by showing where the function is defined (code location with line number) and what `args` are passed to it. 

It's wrapped to a UserError, so the message is like:
```
torch._dynamo.exc.UserError: too many positional arguments.
  func = <code object false_fn at 0x7fe7bfd36fa0, file "/scratch/guangyang/work/pytorch/test/dynamo/test_export.py", line 2514>, args = [<class 'torch.Tensor'>, <class 'torch.Tensor'>], kwargs = {}
```

Reply:
Sorry, what I meant is that returning the raw objects is not helpful. What I was suggesting you do instead is that you grab pre-formatted stack summaries from the tracing context, which should give you the function name / line number as well, but as a standard Python traceback instead of with funny `<code object...>`.

Reply:
I see. That makes sense to me

Reply:
Done